spec/jasper_helpers/links_spec.cr

```diff
@@ -25,7 +25,56 @@ describe JasperHelpers::Links do
     end
 
     it "changes the method when provided" do
-      button_to("Save", "/save", :put).should eq("<form action=\"/save\" class=\"button\" method=\"post\"><input type=\"hidden\" name=\"_method\" id=\"_method\" value=\"put\"/><button type=\"submit\">Save</button></form>")
+      button_to("Save", "/save", :put).should eq button_to_no_block
```

Comment:
I like it but feel that the block should be defined in the same scope for readibility unless it's being called by multiple tests.

Reply:
expected_result =  <<-FORM
   <form action="save" class="button" method="post">
   <input type="hidden" name="_method" id="_method" value="put"/>
   <button type="submit">Save</button>
   </form>
   FORM